control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -255,6 +234,30 @@ func ReconcileKubeAPIServerDeployment(deployment *appsv1.Deployment,
 		},
 	}
 
+	if !strings.Contains(auditConfig.Data[AuditPolicyProfileMapKey], string(configv1.NoneAuditProfileType)) {
```

Comment:
@rtheis  I believe I did try `==` early on, it wasn't matching like how I was expecting so I swapped to strings.Contains. But it may have been in the middle of a debug session. I will swap back on a different branch and re-test to see if it goes through.

Reply:
Tested and it does work. Will update PR to use exact match